src/components/App.tsx

```diff
@@ -64,6 +72,10 @@ const App: React.FC<AppProps> = (props) => {
     return <React.Fragment />
   }
 
+  if (props.waitingForRenderer) {
+    return <LoadingRender />
+  }
+
   if (props.sessionReady) {
```

Comment:
Just for the seek of organization, I would merge waitingForRenderer and sessionReady on a single check

Reply:
Yes! I wasn't sure which would be more stylish. Thanks